pilot/pkg/xds/lds_test.go

```diff
@@ -406,7 +411,9 @@ func TestLDSEnvoyFilterWithWorkloadSelector(t *testing.T) {
 				t.Fatal(err)
 			}
 			defer adsResponse.Close()
-
+			if err := adsResponse.Run(); err != nil {
+				t.Fatal("Error running adsc ", err)
```

Comment:
I think consistent error message will help to identify `ADSC` related issue in logs?
```suggestion
				t.Fatal("ADSC: failed running ", err)
```